examples/signature/dutch_auction_annotated.teal

```diff
@@ -0,0 +1,302 @@
+// GENERATED TEAL         //    PC     PYTEAL PATH                          LINE    PYTEAL
+#pragma version 2         //    (0)    examples/signature/dutch_auction.py  146     Compilation(dutch_auction(), mode=Mode.Signature, version=2).compile(with_sourcemap=True, pcs_in_sourcemap=True, annotate_teal=True, annotate_teal_headers=True, annotate_teal_concise=False)
+global GroupSize          //    (42)                                        136     Global.group_size()
+int 5                     //    (44)                                                Int(5)
+==                        //    (45)                                                Global.group_size() == Int(5)
+bnz main_l6               //    (46)
+global GroupSize          //    (49)                                        137     Global.group_size()
+int 4                     //    (51)                                                Int(4)
+==                        //    (52)                                                Global.group_size() == Int(4)
+bnz main_l5               //    (53)
+global GroupSize          //    (56)                                        138     Global.group_size()
+int 1                     //    (58)                                                Int(1)
+==                        //    (59)                                                Global.group_size() == Int(1)
+bnz main_l4               //    (60)
+err                       //    (63)                                        135     Cond([Global.group_size() == Int(5), bid], [Global.group_size() == Int(4), redeem], [Global.group_size() == Int(1), wrapup])
+main_l4:                  //                                                138     Global.group_size() == Int(1)
+txn RekeyTo               //    (64)                                        118     Txn.rekey_to()
+global ZeroAddress        //    (66)                                                Global.zero_address()
+==                        //    (68)                                                Txn.rekey_to() == Global.zero_address()
+txn FirstValid            //    (69)                                        119     Txn.first_valid()
+int TMPL_START_ROUND      //    (71)                                        15      Tmpl.Int('TMPL_START_ROUND')
```

Comment:
Can now get the PC's for templates

Reply:
pogchamp!

Reply:
Wow. Never knew about this. https://en.wikipedia.org/wiki/PogChamp